api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -529,21 +529,22 @@ function treasuryProposalReducer(
   blockHeight
 ) {
   return {
-    id: `treasury-`.concat(proposal.id),
+    id: `treasury-`.concat(proposal.votes.index),
```

Comment:
`proposal.votes.index` is the unique identifier for the treasury proposal?

Reply:
the same than in Council... there is another one if fact, but not used in Polkassembly. So I follow their advice as they are closely related to W3F

Reply:
> there is another one if fact

another what?

Reply:
I don't understand what you wrote here, can you rephrase?

Reply:
`proposal.votes.index is the unique identifier for the treasury proposal?` ---> this is the main one, yes